src/policy/sft_map.rs

```diff
@@ -128,17 +134,19 @@ mod space_map {
                     assert!(start + bytes <= space_start + MAX_SPACE_EXTENT);
                 }
             }
-            *mut_self.sft.get_unchecked_mut(index) = space;
+
+            //*mut_self.sft.get_unchecked_mut(index) = space;
```

Comment:
Minor: Could you remove the commented out code here? I think there are a couple of other places too.

Reply:
Should be fixed now

Reply:
Thanks. I think there are some other locations where the previous code has been commented out too. Could you remove those lines as well?

Reply:
Should be done now for sure

Reply:
Thanks!